File: engine/engine-model/src/main/scala/tools/packet/character/interaction/GetTradeMeso.scala
package tools.packet.character.interaction

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class GetTradeMeso(private var _number: Byte, private var _meso: Int) extends PacketInput {
  def number: Byte = _number

  def meso: Int = _meso

  override def opcode(): SendOpcode = SendOpcode.PLAYER_INTERACTION
}

File: engine/engine-model/src/main/java/tools/packet/character/interaction/GetTradeMeso.java
package tools.packet.character.interaction;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record GetTradeMeso(Byte number, Integer meso) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PLAYER_INTERACTION;
   }
}